libnavui-maneuver/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <declare-styleable name="MapboxManeuverView">
+    <!-- Defines the background color of MapboxManeuverView primary and secondary instructions as
+     well as step distance remaining and turn icon -->
+    <attr name="maneuverViewBackgroundColor" format="reference|color"/>
+    <!-- Defines the background color of MapboxManeuverView sub instructions and turn icon -->
+    <attr name="subManeuverViewBackgroundColor" format="reference|color"/>
+    <!-- Defines the background color of upcoming maneuvers -->
+    <attr name="upcomingManeuverViewBackgroundColor" format="reference|color"/>
+    <!-- Defines the background color of MapboxLaneGuidance view -->
+    <attr name="laneGuidanceViewBackgroundColor" format="reference|color"/>
+    <!-- Defines the style of maneuver arrow in the lane guidance view -->
+    <attr name="laneGuidanceManeuverArrowStyle" format="reference" />
+  </declare-styleable>
+
+  <declare-styleable name="MapboxTurnIconManeuver">
+    <!-- Defines the color of turn icon -->
+    <attr name="maneuverTurnIcon" format="reference|color"/>
+    <!-- Defines the color of shadow in the turn icon -->
+    <attr name="maneuverTurnIconShadow" format="reference|color"/>
+  </declare-styleable>
+</resources>
```

Comment:
We're styling some elements with builders (like `TurnIconResources`), but we're also exposing some modifiers as XML attributes. Would there be a chance to align the approach and style everything in the runtime with option setters instead of mixing up these concepts?

Reply:
I think it can vary case by case. I don't like the idea of exposing colors via `builders` because it doesn't handle day/night colors automatically. In that case we have to expose separate fields in the builder for day and night based colors. For eg:
`Builder().activeDayColor(RED).activeNightColor(LIGHT_RED).build()`
Even if we do this there is no way for the builder to know what mode the app is running in day/night. If you look at `MapboxManeuverView` and `MapboxTurnIconManeuver` I have exposed public methods that allows you to change the colors in runtime using `@StyleRes`. 
How to handle attributes needs to be a little bit of refactor in a separate PR because each feature is doing them in its own way.